docs/design/chart_format.md

```diff
@@ -394,7 +370,7 @@ The following URI schemes must be supported:
 
 
 #### Special Form Google Storage and S3
-The [Google Storage scheme](https://cloud.google.com/storage/docs/interoperability) (`gs://`)  and the S3 scheme (`s3://`) can also be support. These forms do not follow the URL standard (bucket name is placed in the host field), but their behavior is well-documented.
+The [Google Storage scheme](https://cloud.google.com/storage/docs/interoperability) (`gs://`)  and the S3 scheme (`s3://`) can also be supported. These forms do not follow the URL standard (bucket name is placed in the host field), but their behavior is well-documented.
 
 
 Examples:
```

Comment:
In Appendix B below, we say that we've talked about having a standard (default) chart repository and we then suggest that it be Github based. This wording is problematic on two fronts:
* We already have a standard chart repository named `kubernetes-charts`. We should refer to it explicitly here.
* We have just told the reader why Github based repositories are not optimal. More importantly, we have stripped the Github repository support from the code.